com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -508,7 +508,7 @@ internal static void NetworkBehaviourUpdate()
                     for (int i = 0; i < NetworkManager.Singleton.ConnectedClientsList.Count; i++)
                     {
                         var client = NetworkManager.Singleton.ConnectedClientsList[i];
-                        var spawnedObjs = NetworkSpawnManager.SpawnedObjectsList;
+                        var spawnedObjs = NetworkManager.Singleton.SpawnManager.SpawnedObjectsList;
```

Comment:
```suggestion
                        var spawnedObjs = NetworkManager.SpawnManager.SpawnedObjectsList;
```

Reply:
SpawnManager is not a static property

Reply:
I agree and disagree, but mostly disagree.  The name change is maybe valid (though having small insignificant names for insignificant variables and vice versa is a style choice I actually prefer), but there's also the "when in Rome" principal where we want to make the PR as focused on the actual change at hand, not mix in some minor style edits to keep the signal / noise ratio high.

Reply:
I don't understand why you'd consider this change in a separate PR.

we're changing it from:
```
NetworkSpawnManager.SpawnedObjectsList
```
to
```
NetworkManager.Singleton.SpawnManager.SpawnedObjectsList
```
already.

my only suggestion here is to kill `.Singleton` part as well because there is already `NetworkBehaviour.NetworkManager` property available, so there is no need to access `NetworkManager` over `NetworkManager.Singleton` instead of `NetworkBehaviour.NetworkManager`.

I do not think it should go under a separate PR — we are making a change there already, let's make it even better.

Reply:
This will come in my next PR!
That's where I will convert the whole NetworkObject & Behaviour.